Appendice 3

pablopers opened the pull request:
Ho  riscritto i due blocchi di procedure sulla scorta del documento di Davide.

pablopers committed:
Update appendice-3.rst

edit image in the table (test)

pablopers committed:
Update requirements.txt

update new theme

pablopers committed:
Update conf.py

edit for new theme

pablopers committed:
Update index.rst

pablopers committed:
Update index.rst

pablopers committed:
Update conf.py

pablopers committed:
Update appendice-3.rst

edit procedure

pablopers committed:
Update appendice-3.rst

pablopers committed:
Patch 8 (#7)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update appendice-3.rst

edit image in the table (test)

* Update appendice-3.rst

edit procedure

* Update appendice-3.rst

pablopers committed:
Patch 7 (#6)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

pablopers committed:
Patch 6 (#5)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update appendice-1.rst

edit properties image

pablopers committed:
Patch 5 (#4)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update gestione-dei-documenti.rst

edit properties image

pablopers committed:
Patch 4 (#3)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update pubblicare-un-documento.rst

edit properties image

pablopers committed:
Patch 3 (#2)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update come-partecipare.rst

edit properties image

pablopers committed:
Patch 2 (#1)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

pablopers committed:
Update pubblicare-un-documento.rst

pablopers committed:
Update che-cos-e-docs-italia.rst

pablopers committed:
Update appendice-3.rst

pablopers committed:
Update appendice-3.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update come-partecipare.rst

pablopers committed:
Update appendice-2.rst

alterat commented:
@pablopers Kudos per il tuo contributo! Lascio un attimo in attesa quest'ultima PR perché approvarla ora significherebbe visualizzare in maniera errata le procedure nella vecchia versione RTD del documento. 

Conto di fare merge al più presto, non appena passeremo alla nuova piattaforma.

pablopers committed:
Patch 2 (#12)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

* Update come-partecipare.rst

edit properties image

* Update pubblicare-un-documento.rst

edit properties image

* Update gestione-dei-documenti.rst

edit properties image

* Update appendice-1.rst

edit properties image

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

* Update che-cos-e-docs-italia.rst

* Update pubblicare-un-documento.rst

* Modifiche cosmetiche

* Modifiche cosmetiche

* Modifiche cosmetiche

* Rimosso titolo doppio

* Update pubblicare-un-documento.rst

new class procedure

* Update scrivere-un-documento.rst (#10)

add two class procedure

* Update requirements.txt

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

pablopers committed:
Patch 1 (#11)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

* Update come-partecipare.rst

edit properties image

* Update pubblicare-un-documento.rst

edit properties image

* Update gestione-dei-documenti.rst

edit properties image

* Update appendice-1.rst

edit properties image

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

* Update che-cos-e-docs-italia.rst

* Update pubblicare-un-documento.rst

* Modifiche cosmetiche

* Modifiche cosmetiche

* Modifiche cosmetiche

* Rimosso titolo doppio

* Update come-partecipare.rst

corretto sec rotto

* Update pubblicare-un-documento.rst (#8)

corretti sec rotti

pablopers committed:
Patch 2 (#13)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

* Update come-partecipare.rst

edit properties image

* Update pubblicare-un-documento.rst

edit properties image

* Update gestione-dei-documenti.rst

edit properties image

* Update appendice-1.rst

edit properties image

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

* Update che-cos-e-docs-italia.rst

* Update pubblicare-un-documento.rst

* Modifiche cosmetiche

* Modifiche cosmetiche

* Modifiche cosmetiche

* Rimosso titolo doppio

* Update pubblicare-un-documento.rst

new class procedure

* Update scrivere-un-documento.rst (#10)

add two class procedure

* Update requirements.txt

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

* Update conf.py

* class procedure (#9)

* Update appendice-3.rst

edit image in the table (test)

* Update appendice-3.rst

edit procedure

* Update appendice-3.rst

* Update appendice-3.rst

pablopers committed:
procedure 1 (#14)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

* Update come-partecipare.rst

edit properties image

* Update pubblicare-un-documento.rst

edit properties image

* Update gestione-dei-documenti.rst

edit properties image

* Update appendice-1.rst

edit properties image

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

* Update che-cos-e-docs-italia.rst

* Update pubblicare-un-documento.rst

* Modifiche cosmetiche

* Modifiche cosmetiche

* Modifiche cosmetiche

* Rimosso titolo doppio

* Update come-partecipare.rst

corretto sec rotto

* Update pubblicare-un-documento.rst (#8)

corretti sec rotti

* Update pubblicare-un-documento.rst

edit procedure

pablopers committed:
procedure 2 (#15)

* Update scrivere-un-documento.rst

Aggiunta del tipo di file ODT per il convertitore web

* Aggiunta virgola

* Aggiunti campi obbligatori

* Aggiunti campi obbligatori nella tabella

* Segnalati campi obbligatori yml

* Aggiunto document_settings

* Aggiunti numeri sezione

* Test link figure

* Update che-cos-e-docs-italia.rst

edit proprieties image

* Update che-cos-e-docs-italia.rst

* Update come-partecipare.rst

edit properties image

* Update pubblicare-un-documento.rst

edit properties image

* Update gestione-dei-documenti.rst

edit properties image

* Update appendice-1.rst

edit properties image

* Update appendice-2.rst

edit properties image

* Update appendice-2.rst

* Update appendice-2.rst

* Update che-cos-e-docs-italia.rst

* Update pubblicare-un-documento.rst

* Modifiche cosmetiche

* Modifiche cosmetiche

* Modifiche cosmetiche

* Rimosso titolo doppio

* Update come-partecipare.rst

corretto sec rotto

* Update pubblicare-un-documento.rst (#8)

corretti sec rotti

* Update scrivere-un-documento.rst

edit syntax procedure

pablopers committed:
Update appendice-3.rst

edit procedure and fix numbered

pablopers committed:
Merge branch 'master' into patch-8
